### ISSUE TITLE
Carbon selection style and color

### ISSUE THREAD

**Issue by abhijit945** [cerner/carbon-graphs#7]
UX to investigate the most appropriate selection styling and color for all data points and graph types. 

Include before and after colors.

**Comment by laurenbock** [comment]
Use Terra selection colors. The selection should surround the icon, not just be centered within the gantt row for gantt charts.

**Comment by sdadn** [comment]
@laurenbock Can you elaborate on using Terra colors?

**Comment by jeremyfuksa** [comment]
Triaging old issues. Has this been resolved outside of this discussion thread?

**Comment by laurenbock** [comment]
@jeremyfuksa I missed this one - no it has not been resolved yet. We need to darken the selection state colors in Carbon. I proposed using Terra's colors so that it was consistent and would pass contrast. However, I know there are more than one set of selection colors right now in Terra. Is there one that you'd recommend that is the safest to use?

**Comment by jeremyfuksa** [comment]
It's difficult to give a definitive answer since the display criteria are different given the context of the component. However, from a contrast _and_ consistency standpoint, the focus state of pagination might be a suitable style: https://zpl.io/Vq1GqkJ

**Comment by laurenbock** [comment]
@jeremyfuksa  I do like that style as it is similar to Carbon today, but I want to check because it doesn't look like the blue passes contrast - is this ok with accessibility? 
![image](https://user-images.githubusercontent.com/57807487/87503215-444bd280-c629-11ea-9903-e99234cdcb94.png)
![image](https://user-images.githubusercontent.com/57807487/87503232-4f9efe00-c629-11ea-8f34-81fe6cb84a28.png)

**Comment by jeremyfuksa** [comment]
It wouldn't. Pagination has a number of accessibility issues that are being worked on, it turns out. So my recommendation is not good. The closest color would be `#1B9ADF`, which is just off @blue60, since it doesn't pass WCAG standards either.

TL;DR: I need to find time to return to my color study that tweaks our color palette for better accessibility.

**Comment by erinplourde** [comment]
@laurenbock @jeremyfuksa  So apparently way back in the day I did some update on the selection color - take a look and let me know your thoughts around it. I can continue the exploration if needed.

[SelectionState.pdf](https://github.com/cerner/carbon-graphs/files/5119786/SelectionState.pdf)

**Comment by jeremyfuksa** [comment]
@erinplourde I like the approach.

**Comment by laurenbock** [comment]
that's really helpful to see, @erinplourde! 

@jeremyfuksa do you know the answers to any of the questions in red? I do not - I wonder if Scott would be best to ask (i know he is out on sabbatical atm, though)

**Comment by jeremyfuksa** [comment]
> Does the selection color need to meet 3:1 to more than just the white background?

Since data points could potentially sit on axis lines, it should. The example in the PDF fails at 2.47:1 on the gray line.

> Are the colors bright enough to differential between our palette blues? Currently the new selection blue is close to the dark blue and could cause some worry with bubble and gantt charts - This needs to be looked at.

I think so. So, in order to reach contrast, tweaking the saturation just a bit should get you there. `#148bcc` looks pretty good and meets contrast. It should also be saturated enough to not compete with the dark blue.

> Does our selection colors match with terra selection colors?

They aren't exact at the moment, but they are close. I've been working on unifying the palette a little more on and off over the past year. Once that gets nailed down, we can enhance Carbon's palette to be aligned with whatever we come up with.,

**Comment by joelsamthomas** [comment]
H, @jeremyfuksa / @sdadn, Is there an ETA on this issue. We have a consumption Jira being logged for this.

Thanks,
Joel

**Comment by sdadn** [comment]
@joelsamthomas Sorry, I don't think we have any update on this issue right now as we just migrated Carbon to a [new repository](https://github.com/cerner/terra-graphs) and are currently catching up on fixing bugs.

**Comment by jeremyfuksa** [comment]
> @joelsamthomas Sorry, I don't think we have any update on this issue right now as we just migrated Carbon to a [new repository](https://github.com/cerner/terra-graphs) and are currently catching up on fixing bugs.

Let's move this issue over to the new repo and use my suggested color of `#148bcc` as the fix.
